### PR TITLE
Remove code-blocks

### PR DESCRIPTION
**Pull request by MongoCaleb**
replace code-blocks with snippets (incl. codeblock and Codeblock, too)

### CODE REVIEW

File: src/bluehawk/actions/snip.test.ts [hunk 1]
```diff
@@ -280,15 +280,15 @@ line 9
...
+      "test.snippet.foo.js.snippet.rst",
     ]);
 
     const fileContents = await System.fs.readFile(
-      Path.join(outputPath, "test.codeblock.foo.js.code-block.rst"),
+      Path.join(outputPath, "test.snippet.foo.js.snippet.rst"),
       "utf8"
     );
-    expect(fileContents).toStrictEqual(`.. code-block:: javascript
+    expect(fileContents).toStrictEqual(`.. snippet:: javascript
```
Comment (cbush) [inline src/bluehawk/actions/snip.test.ts:291]: In this case it still needs to be code-block for rST's sake

**Review by cbush — CHANGES_REQUESTED** [review]

File: src/bluehawk/actions/snip.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ export const formatInRst = async (
     ? rstLanguageMap.get(document.extension)
     : "text";
 
-  const rstHeader = ".. code-block::";
+  const rstHeader = ".. snippet::";
```
Comment (mongodben) [inline src/bluehawk/actions/snip.ts:86]: please revert this

File: src/bluehawk/actions/snip.test.ts [hunk 3]
```diff
@@ -51,16 +51,16 @@ describe("snip", () => {
...
+      "test.snippet.foo.js.snippet.rst",
     ]);
 
     const rstFileContents = await System.fs.readFile(
-      Path.join(outputPath, "test.codeblock.foo.js.code-block.rst"),
+      Path.join(outputPath, "test.snippet.foo.js.snippet.rst"),
       "utf8"
     );
-    expect(rstFileContents).toStrictEqual(`.. code-block:: javascript
+    expect(rstFileContents).toStrictEqual(`.. snippet:: javascript
```
Comment (mongodben) [inline src/bluehawk/actions/snip.test.ts:63]: revert pls. see above

File: src/bluehawk/actions/snip.test.ts (hunk 1 above)
Comment (mongodben) [inline src/bluehawk/actions/snip.test.ts:291]: revert pls

File: src/bluehawk/actions/snip.test.ts [hunk 4]
```diff
@@ -514,12 +514,12 @@ struct ContentView: SwiftUI.App {
...
       Path.join(
         outputPathSync,
-        "test.codeblock.content-view.swift.code-block.rst"
+        "test.snippet.content-view.swift.snippet.rst"
       ),
       "utf8"
     );
     expect(fileContentsSync).toStrictEqual(
-      `.. code-block:: swift
+      `.. snippet:: swift
```
Comment (mongodben) [inline src/bluehawk/actions/snip.test.ts:522]: revert pls

File: src/bluehawk/actions/snip.test.ts [hunk 5]
```diff
@@ -542,15 +542,15 @@ struct ContentView: SwiftUI.App {
...
+        "test.snippet.content-view.swift.snippet.rst"
       ),
       "utf8"
     );
     // TODO: Expect this not to emphasize lines, since the :emphasize: tag was
     // completely in the other state
     /*
   expect(fileContentsLocal).toStrictEqual(
-    `.. code-block:: swift
+    `.. snippet:: swift
```
Comment (mongodben) [inline src/bluehawk/actions/snip.test.ts:553]: revert pls

**Review by cbush — APPROVED** [review]

**Pull request merged**